main.go

```diff
@@ -90,105 +111,26 @@ func main() {
 	livenessMonitor.Enable()
 	readinessMonitor.Enable()
```

Comment:
note: Those monitors are never disabled. Probably, we should use `defer monitor.Disable()`